pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -413,7 +424,7 @@ const decommissionedNodesTableData = createSelector(
   partitionedStatuses,
   nodesSummarySelector,
   (statuses, nodesSummary): DecommissionedNodeStatusRow[] => {
-    const decommissionedStatuses = statuses.decommissioned || [];
+    const decommissionedStatuses = statuses.live || [];
```

Comment:
I think this modification is incorrect and it should remain `decommissioned`. Can you take another look?

Maybe add a test to check :)

Reply:
🤦‍♂️oh, that's a terrible result of manual testing where I used "live" nodes to emulate some data in Decommissioned nodes list, and haven't reverted code back.
It is a good point to add tests on these selectors.